tests/core/full_node/test_full_node.py

```diff
@@ -947,8 +947,8 @@ async def test_new_transaction_and_mempool(self, wallet_nodes, self_hostname):
                 if force_high_fee:
                     successful_bundle = spend_bundle
             else:
-                assert full_node_1.full_node.mempool_manager.mempool.at_full_capacity(5000000 * group_size)
-                assert full_node_1.full_node.mempool_manager.mempool.get_min_fee_rate(5000000 * group_size) > 0
```

Comment:
this change may be a bit mysterious. This test did not used to run in `HARD_FORK_2_0` mode, but since I removed some restrictions (along with `SOFT_FORK4`) this started running (and failing) in hard fork mode.

I tracked it down to the cost of the transaction we're trying to add to the mempool cost more than `5000000 * group_size`. So, just because we can't fit that TX in the mempool doesn't mean it's at capacity for this size.